release-notes.md

```diff
@@ -8,6 +8,43 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 Please take a moment to fill out a [short-survey](https://www.surveymonkey.com/r/AEPDocs) on how we can better assist you with enabling Adobe Experience Cloud solutions and services on your mobile apps.
 
+## January 13, 2020
+
+The following updates were made in this release:
+
+**iOS Griffon 1.0.3**
+
+* Griffon bridge and Griffon SDK unified. Setup steps are slightly different now. See [here](https://aep-sdks.gitbook.io/docs/beta/project-griffon/set-up-project-griffon) for updated documentation.
+* Adobe Analytics debug flag now enabled on Griffon session start and disabled on session stop.
+* Client-side UI now logs location service events (entry/exit) events.
+* Updated for iOS 13 compatibility.
+* Fixed a crash of LIBDISPATCH.
+
+**Android Griffon 1.1.0**
+* Griffon bridge and Griffon SDK unified. Setup steps are slightly different now. See [here](https://aep-sdks.gitbook.io/docs/beta/project-griffon/set-up-project-griffon) for updated documentation.
+* Adobe Analytics debug flag now enabled on Griffon session start and disabled on session stop.
+* Client-side UI now logs location service events (entry/exit) events
+* Modified client-ui to include client-side logging capabilities.
+* Fixed an issue for Android API versions 28 or later where multiple WebViews cannot share the same data directory.
+* Added generic exception handling for exceptions that can occur when WebViews are accessed while OS is updating Chrome.
+
+The Project Griffon web UI now has new views specifically for users trying to inspect and improve Adobe Analytics and Location Service (Places) implementations.
```

Comment:
Please reword to:
"The Project Griffon web UI now has views for users who are trying to inspect and improve Adobe Analytics and Location Service (Places) implementations."

(When Ivan confirms the name, we should change "Location Service".)